### ISSUE TITLE
Remove italics in ipython

### ISSUE THREAD

**Issue by dehlong** [catppuccin/python#60]
Hello,
I would like to use ipython with this theme but by default some comments are with italics:
![image](https://github.com/user-attachments/assets/89e69da7-1a54-437b-bd58-add107ca2467)
Can I somehow disable it?

**Comment by backwardspy** [comment]
i will have to look into it - i don't know if ipython themes support options. if they don't, we could provide no-italics variants of each theme perhaps.

**Comment by dehlong** [comment]
Yeah, thanks. catppuccin by default uses italics (even in vscode) but there I can change it editor settings. I do not think it is possible here,
